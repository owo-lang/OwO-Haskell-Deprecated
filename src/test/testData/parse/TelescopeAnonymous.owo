module TelescopeAnonymous where

  id : {A : Type} -> A -> A
