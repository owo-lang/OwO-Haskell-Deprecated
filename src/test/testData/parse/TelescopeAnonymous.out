File type: owo
Top module name: .TelescopeAnonymous
 type signature id (35 3 3) (37 3 5)
  no pragmas
  telescope binding A (41 3 9) (42 3 10)
  { }
  binding type
   named reference Type (45 3 13) (49 3 17)
  return type
   telescope binding _ (54 3 22) (55 3 23)
   ( )
   binding type
    named reference A (54 3 22) (55 3 23)
   return type
    named reference A (59 3 27) (60 3 28)
